qa/L1_jupyter_plugins/test.sh

```diff
@@ -2,18 +2,16 @@
 # used pip packages
 pip_packages="jupyter matplotlib opencv-python mxnet-cu90 tensorflow-gpu torchvision torch"
 
+# We need cmake to run the custom plugin notebook + ffmpeg and wget for video example
 apt-get update
-apt-get install -y ffmpeg
+apt-get install -y wget ffmpeg
+apt-get install -y --no-install-recommends cmake
```

Comment:
not sure if the no-install-recommends was necessary here, so I suggest you install cmake in line 7 as well

Reply:
I'm not sure neither:
https://github.com/NVIDIA/DALI/commit/d6d9b470f8df31a8cf5427fc329b444ec44e7f1a
:stuck_out_tongue:

Reply:
Updated